source/includes/_invoice-payment.md

```diff
@@ -828,9 +574,6 @@ invoicePaymentApi.create_chargeback_reversal(payment_id,
    "auditLogs":[]
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?